packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -268,7 +268,8 @@ export const generateDefaultDeviceMenuProps = (
       // A side effect of this workaround is that the context menu stays open when window is resized, and may
       // get detached from original target visually. That bug is preferable to the bug when this value is not set -
       // The Callout (frequently) gets dismissed automatically.
-      preventDismissOnResize: true
+      preventDismissOnResize: true,
```

Comment:
Please add a comment explaining why we need to prevent menu dismissal on scroll. Essentially include the context from the bug you're fixing in the comment so that somebody reading the code understands why we set this flag.